docs/sdks/dotnet/GETTING_STARTED.md

```diff
@@ -1,44 +1,47 @@
 ## Getting Started
 
 ### Initialize & Make API Request
-Once you add the dependencies, its extremely easy to get started with the SDK; All you need to do is import the package in your code, set your Appwrite credentials, and start making API calls. Below is a simple example:
+Once you have installed the package, it is extremely easy to get started with the SDK; all you need to do is import the package in your code, set your Appwrite credentials, and start making API calls. Below is a simple example:
 
 ```csharp
 using Appwrite;
+using Appwrite.Services;
+using Appwrite.Models;
 
-static async Task Main(string[] args)
-{
-  var client = Client();
+var client = new Client()
+  .SetEndpoint("http://cloud.appwrite.io/v1")  // Make sure your endpoint is accessible
+  .SetProject("5ff3379a01d25")                 // Your project ID
+  .SetKey("cd868db89")                         // Your secret API key
+  .SetSelfSigned();                            // Use only on dev mode with a self-signed SSL cert
```

Comment:
I don't think you should have this here if it's pointed to cloud.

Reply:
Should we do that, or maybe we can update the comment to mention that it should only be used if the user has a self-hosted instance without a domain pointed to it?

Reply:
Our attitude at the moment is cloud first, so make this copy pasteable for cloud.

Maybe, we can split this into cloud and self-hosted (we should do for all SDKs) but you can leave a comment in experience channel later and I'll make it happen following ur example here ;)

Reply:
Made the update :)